src/db/add_package.rs

```diff
@@ -63,7 +63,7 @@ pub(crate) fn add_package_into_database(conn: &Connection,
                                          &metadata_pkg.version,
                                          &release_time,
                                          &dependencies.to_json(),
-                                         &metadata_pkg.targets[0].name.replace("-", "_"),
+                                         &metadata_pkg.library_name().unwrap(),
```

Comment:
Yes, I tested this locally with `ripgrep 11.0.2` and it will crash the builder:

```
2019/11/25 12:58:40 [DEBUG] cratesfyi::db::add_package: Adding package into database
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:378:21
  15: cratesfyi::db::add_package::add_package_into_database
  16: rustwide::build::BuildBuilder::run
  17: cratesfyi::docbuilder::rustwide_builder::RustwideBuilder::build_package
  18: cratesfyi::main
```

Reply:
For the binary case you could probably use the existing code:

```suggestion
                                         &metadata_pkg.library_name().unwrap_or_else(|| {
                                metadata_pkg.targets[0].name.replace("-", "_")
                              }),
```

Reply:
...totally forgot about that. We really need a test suite `:/`